package.json

```diff
@@ -53,6 +53,10 @@
     "webpack-utf8-bom": "^1.3.0"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.4",
+    "@emotion/styled": "^11.10.4",
+    "@mui/icons-material": "^5.10.9",
+    "@mui/material": "^5.10.11",
```

Comment:
We won't need any of these new dependencies, I'll show you down below how to get `styled` at Webchat.

Reply:
@murilovmachado It's done.